common/lib/mandrill_client/client.py

```diff
@@ -38,11 +38,12 @@ class MandrillClient(object):
     REFERRAL_FOLLOW_UP_EMAIL = 'referred-learner-follow-up'
     REFERRAL_SOCIAL_IMPACT_TOOLKIT = 'social-impact-toolkit'
     MINI_COURSE_ENROLMENT = 'mini-course-enrolment'
+    SEND_ACTION_PLAN = 'send-action-plan'
 
     def __init__(self):
         self.mandrill_client = mandrill.Mandrill(settings.MANDRILL_API_KEY)
 
-    def send_mail(self, template_name, user_email, context, attachments=[], subject=None):
+    def send_mail(self, template_name, user_email, context, attachments=[], subject=None, receiver_emails=[], sender_email=None, images=[]):
```

Comment:
can we use **kwargs here

Reply:
new method has less arguments with meaningfull names